modules/settings/classes/settings.php

```diff
@@ -17,7 +18,7 @@ class Settings {
 	 * @param string $option_name
 	 * @return mixed
 	 */
-	public static function get( string $option_name ) {
+	public static function get( string $option_name ) :mixed {
```

Comment:
mixed type is not supported in php 7.4 please remoev and make sure your ide is setup for php 7.4

Reply:
Updated @bainternet